### PR TITLE
Device

### PR DESCRIPTION
**Pull request by Enanodr**
## Summary

- Integrated with Devise for user session managment (log in, sign in and log out)
- Added log out and log in on the welcome, show article and list article pages
- Added a request to log in, in order to create or destroy articles

## TrelloCard
https://trello.com/c/ti0JI29O/9-instalar-y-configurar-devise

### login/logout at index
image

### login/logout at arcticle listing on the top
image

### login/logout at article display
image

### login form
image

### CODE REVIEW

**Comment by epintos** [comment]
@Enanodr It would be nice to have Screenshots in this PR also

File: blog/app/views/devise/confirmations/new.html.slim [hunk 1]
```diff
@@ -0,0 +1,11 @@
+h2
```
Comment (epintos) [inline blog/app/views/devise/confirmations/new.html.slim]: All this views are not really necessary if you don't need to modify them, yo could delete them

File: blog/test/fixtures/users.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (epintos) [inline blog/test/fixtures/users.yml]: Same comment as before about the test folder

**Comment by epintos** [comment]
:+1:

**Comment by Enanodr** [comment]
Merged manually to master

**Pull request closed without merging**